build/patches/experimental-user-scripts-support.patch

```diff
@@ -0,0 +1,10299 @@
+From: uazo <uazo@users.noreply.github.com>
+Date: Sat, 20 Mar 2021 09:37:12 +0000
+Subject: Experimental user scripts support preview
+
```

Comment:
We need to copy a lot of the information from the PR in the commit body here, can you please start this? You have already a good start in the PR description. Maybe add the updates that you did; no need to add any discussion, just describe how it is.

Reply:
I thought I will add a README.md which I have yet to prepare

Reply:
I prefer the patch description because there are too many files and folders in the Chromium code base.

Reply:
however, adding the readme in the components is common practice in the codebase and I am afraid that the comment will not be that long

Reply:
Sure, then mention in the patch that there is a README